src/docs/stan-reference/functions.tex

```diff
@@ -3131,15 +3140,21 @@ \subsubsection{QR Decomposition}\label{QR-decomposition}
 %
 \end{description}
 %
+The thin QR decomposition is generally preferable and will consume much
```

Comment:
Isn't it always preferred? Your choice, but shouldn't we just discourage folks from ever regular QRing in Stan, and then use the thin QR for the tupley QR whenever that happens? Is there any downside?